sympy/matrices/expressions/matpow.py

```diff
@@ -4,18 +4,19 @@
 from sympy.core.sympify import _sympify
 from sympy.core.compatibility import range
 from sympy.matrices import MatrixBase
-from sympy.core import S
+from sympy.core import S, Symbol
 
 
 class MatPow(MatrixExpr):
 
-    def __new__(cls, base, exp):
+    def __new__(cls, base, exp, evaluate=False):
```

Comment:
We usually have
```python
from sympy.core.evaluate import global_evaluate

[...]

def __new__(cls, [...] , **kwargs):
    evaluate = kwargs.get('evaluate', global_evaluate[0])
```

We could also create a new global parameter to determine whether matrix powers are evaluated.